<launch>
  <node pkg="multi_object_tracking" type="multi_object_tracking" name="multi_object_tracking"
        output="screen">

    <param name="m_merge_close_hypotheses_distance" value="0.60" />
    <param name="m_max_mahalanobis_distance" value="10.0" />
    <param name="m_world_frame" value="odom" />
    <param name="m_born_time_threshold" value="0.65" />
    <param name="m_future_time" value="0.05" />

  </node>
</launch>
